certification/internal/policy/operator/deployable_by_olm.go

```diff
@@ -178,30 +182,37 @@ func (p *DeployableByOlmCheck) installedCSV(operatorData OperatorData, openshift
 	defer cancel()
 
 	installedCSVDone := make(chan string, 1)
+	contextTimeOut := make(chan error, 1)
 
+	var installedCSV string
 	// query API server for the installed CSV field of the created subscription
-	go func() {
+	go func(ctx context.Context) string {
 		defer close(installedCSVDone)
```

Comment:
You don't need to pass it in. Just create it within this scope.

Reply:
Passing context as argument is preferred (https://go.dev/blog/context-and-structs). This way I can explicitly attach a function/goroutine to the context.

Reply:
I believe both are working. I believe passing `context` as an arg will explicitly set the boundaries for the code that are executed within the context.